CenterForOpenScience/osf.io
# [ENG-2464] Fix license admin listing

## Purpose

Touch up the default license listing to be more user friendly.

## Changes

- simple upgrade of old `__unicode__` to `__str__`

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify you can update default licenses for Preprint Providers


## Documentation

🐞 fix, no docs/

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-2464
